python_frank_energie/frank_energie.py

```
@@ -19,251 +45,999 @@ class FrankEnergie:
-                }
-            """,
-            "operationName": "Login",
-            "variables": {"email": username, "password": password},
+                """,
+                "operationName": "Login",
+                "variables": {"email": username, "password": password},
+            }
+
+            response = await self._query(query)
+            auth_data = None
+            if not response is None:
```

Ruff

Test for object identity should be `is not`

Convert to `is not`

(E714)